src/utils_request.js

```diff
@@ -281,7 +28,7 @@ function decompress(response) {
  * Currently, the function sends requests the same way as Firefox web browser does.
  * In the future, it might add support for other browsers too.
  *
- * Internally, the function uses `requestBetter()` function to perform the request.
+ * Internally, the function uses `@apify/httpRequest` function to perform the request.
  * All `options` not recognized by this function are passed to it,
```

Comment:
Can we add a link to NPM package page with the docs? Also, it's not `@apify/httpRequest` function, but `httpRequest` in the `@apify/http-request` package.